protocol/x/clob/keeper/liquidations.go

```diff
@@ -164,6 +154,12 @@ func (k Keeper) MaybeGetLiquidationOrder(
 		return nil, types.ErrSubaccountNotLiquidatable
 	}
 
+	defer telemetry.ModuleMeasureSince(
+		types.ModuleName,
+		time.Now(),
+		metrics.MaybeGetLiquidationOrder,
+	)
```

Comment:
this naming be misleading because you are leaving out the `IsLiquidatable` call above which presumably is a relatively "expensive" call. You are reading from state which is a lot of marshaling and unmarshaling, then looking through every position and every perpetual and every price that's relevant. Then doing calculations using `big.Int` and `big.Rat` types